test/issue-strategies/dispatcher-test.js

```diff
@@ -0,0 +1,17 @@
+import * as dispatcherStrat from '../../src/issue-strategies/dispatcher.js';
+import DummyJira from '../dummy-jira.js';
+
+describe('Dispatcher/Subtask Strategy Apply Tests', () =>
+{
+	let dummyJira;
+
+  beforeEach(function() {
+     dummyJira = new DummyJira();
+  });
+  describe('Not ok to commit against', () =>
+	{
+		it('should fail because you can\'t commit against dispatcher', ()=>
+				dispatcherStrat.apply(dummyJira.issues.Dispatcher1, dummyJira).should.eventually.be.rejectedWith(Error, /Cannot commit against a Dispatcher/));
+	});
+});
+
```

Comment:
Please mind your spacing here. Indentation should be with two spaces, not one, and javascript practice is to not newline the opening curly brace like you would in C#.